src/cryptsetup/cryptsetup-generator.c

```diff
@@ -170,6 +185,10 @@ static int create_disk(
         swap = fstab_test_option(options, "swap\0");
         netdev = fstab_test_option(options, "_netdev\0");
 
+        keyfile_timeout = fstab_filter_options(options, "keyfile-timeout\0", NULL, &keydev_timeout, NULL);
+        if (keyfile_timeout < 0)
+            return log_error_errno(keyfile_timeout, "Failed to parse keyfile-timeout= option value: %m");
```

Comment:
borked indentaion...

this is superconfusing, that this returns a keyfile_timeout *and* a keydev_timeout variable